### ISSUE TITLE
Missing Class: Google Http MediaFileUpload while uploading Image(s) manually

### ISSUE THREAD

**Issue by A-Matt** [udx/wp-stateless#642]
Recently after updating to Version 3.2.2 we've been unable to Upload new pictures to our website through the normal Media uploader. We've been unable to reproduce this on the dozen or so other websites we're using which have yet to update yet.

The Server is reporting the following error:
```log
[Wed Jun 08 14:11:19.027388 2022] [proxy_fcgi:error] [pid 12724:tid 140497582491392] [client 172.69.79.172:44512] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught Error: Class 'Google\\Google_Http_MediaFileUpload' not found in /var/www/vhosts/WEBSITE/httpdocs/wp-content/plugins/wp-stateless/lib/classes/class-gs-client.php:234
Stack trace:
#0 /var/www/vhosts/WEBSITE/httpdocs/wp-content/plugins/wp-stateless/lib/classes/class-utility.php(352): wpCloud\\StatelessMedia\\GS_Client->add_media()
#1 /var/www/vhosts/WEBSITE/httpdocs/wp-includes/class-wp-hook.php(307): wpCloud\\StatelessMedia\\Utility::add_media()
#2 /var/www/vhosts/WEBSITE/httpdocs/wp-includes/plugin.php(189): WP_Hook->apply_filters()
#3 /var/www/vhosts/WEBSITE/httpdocs/wp-includes/post.php(6597): apply_filters()
#4 /var/www/vhosts/WEBSITE/httpdocs/wp-admin/includes/image.php(351): wp_update_attachment_metadata()
#5 /var/www/vhosts/WEBSITE/httpdocs/wp-admin/includes/image.php(161): wp_create_image_subsizes()
#6 /var/www/vhosts/WEBSITE/httpdocs/wp-admin/includes/ajax-actions.php(2469): wp_update_image_su...', referer: https://WEBSITE/wp-admin/media-new.php
```

To confirm that this wasn't a bad install as we have uninstalled and re-installed from WordPress' Plugin Store/Shop.
All other Websites we are using are on 3.1.1

**Comment by planv** [comment]
Fixed.
Will be released in version 3.2.3

**Comment by planv** [comment]
Released. Closing the ticket.
Please let us know if you have any further questions
